### PR TITLE
Add Reflect.hasMetadata to support InversifyJS 

### PR DESCRIPTION
**Pull request by sublimator**
Note that there are no real tests for hasOwnMetadata as the tests for getOwnMetadata "cover" things. Likewise here.

### CODE REVIEW

**Comment by sublimator** [comment]
@abraham  ? :) :) :) ??

File: src/index.test.ts [hunk 1]
```diff
@@ -25,6 +25,11 @@ test('hasOwnMetadata', () => {
   expect(Reflect.hasOwnMetadata).toEqual(Reflection.hasOwnMetadata);
 });
 
+test('hasMetadata', () => {
```
Comment (sublimator) [inline src/index.test.ts]: Did at least add this test :)

**Comment by sublimator** [comment]
@abraham 
Any love good sir ?

**Comment by sublimator** [comment]
@zaynyatyi 
You need this too
?

**Comment by zaynyatyi** [comment]
Yes, for now i applied your change locally :) would be happy if that would be merged

**Comment by abraham** [comment]
Thanks, sorry about the delay.

**Comment by zaynyatyi** [comment]
<3 thank guys, that's great news

**Comment by sublimator** [comment]
woohOOOO!!!

thanks kind sir

**Comment by abraham** [comment]
v0.7.0 has been published.

**Pull request merged**